lib/util/cache/package/file.ts

```diff
@@ -99,6 +99,7 @@ export async function cleanup(): Promise<void> {
         deletedCount += 1;
       }
     }
+    await cacache.verify(cacheFileName);
```

Comment:
```suggestion
    logger.debug("Verifying and cleaning cache.", { cacheFileName });
    await cacache.verify(cacheFileName);
```

We should add a debug log if it really can take 11min to complete.